src/openrct2/peep/Peep.cpp

```diff
@@ -3241,7 +3241,7 @@ rct_string_id get_real_name_string_id_from_id(uint32_t id)
     return dx;
 }
 
-static int32_t peep_compare(const void* sprite_index_a, const void* sprite_index_b)
+int32_t peep_compare(const void* sprite_index_a, const void* sprite_index_b)
```

Comment:
Why? I'm assuming there are more changes to follow

Reply:
? Not sure what you mean. I need this function in the guest list that's why it's no longer static.

Reply:
I misunderstood, my bad.